targets/csharp/source/source/Uunit/tests/PlayFabApiTest.cs

```diff
@@ -537,10 +480,169 @@ private void GetObjects2Continued(PlayFabResult<GetObjectsResponse> result, UUni
             if (!int.TryParse(result.Result.Objects[TEST_DATA_KEY].EscapedDataObject, out int actualValue))
                 actualValue = -1000;
             testContext.IntEquals(_testInteger, actualValue, "Failed: " + _testInteger + "!=" + actualValue + ", Returned json: " + result.Result.Objects[TEST_DATA_KEY].EscapedDataObject);
-
             testContext.EndTest(UUnitFinishState.PASSED, null);
         }
 #endif
+        /// <summary>
+        /// CLIENT API
+        /// Test that Multiple login can be done
+        /// </summary>
+        [UUnitTest]
+        public void MultipleLogin(UUnitTestContext testContext)
+        {
+            // If the setup failed to log in a user, we need to create one.
+            var loginRequest = new LoginWithCustomIDRequest
+            {
+                TitleId = PlayFabSettings.TitleId,
+                CustomId = PlayFabSettings.BuildIdentifier,
+                CreateAccount = true,
+            };
+            var loginRequest2 = new LoginWithCustomIDRequest
+            {
+                TitleId = PlayFabSettings.TitleId,
+                CustomId = PlayFabSettings.BuildIdentifier,
+                CreateAccount = true,
+            };
+
+            var loginTask = PlayFabClientAPI.LoginWithCustomIDAsync(loginRequest, null, testTitleData.extraHeaders).Result;
+            var loginTask2 = PlayFabClientAPI.LoginWithCustomIDAsync(loginRequest2, null, testTitleData.extraHeaders).Result;
+
+            testContext.NotNull(loginTask.Result, "Login Result is null for loginRequest");
+            testContext.NotNull(loginTask2.Result, "Login Result is null for loginRequest2");
+            testContext.IsNull(loginTask.Error, "Login error occured for loginRequest: " + loginTask.Error?.ErrorMessage ?? string.Empty);
+            testContext.IsNull(loginTask2.Error, "Login error occured for loginRequest2: " + loginTask2.Error?.ErrorMessage ?? string.Empty);
+            testContext.NotNull(loginTask.Result.AuthenticationContext, "AuthenticationContext is not set for loginRequest");
+            testContext.NotNull(loginTask2.Result.AuthenticationContext, "AuthenticationContext is not set for loginRequest2");
+
+            if (string.Equals(loginTask.Result.AuthenticationContext.ClientSessionTicket, loginTask2.Result.AuthenticationContext.ClientSessionTicket))
+            {
+                testContext.Fail("Multiple Login Failed AuthenticationContexts are same");
+            }
+            testContext.EndTest(UUnitFinishState.PASSED, null);
```

Comment:
At the end of this test, before passing, please save off AuthenticationContext1 and AuthenticationContext2 into private test class varaibles, and then in the next test...